src/client/components/terminal.ts

```diff
@@ -349,18 +349,27 @@ export class TerminalView extends LitElement {
   }
 
   protected firstUpdated(props: PropertyValues): void {
+    console.log('got to first updated cb!!')
```

Comment:
Still required?

Reply:
Oh yeah, not needed.

I need to get better about this haha